core/transaction_files/transactiontree.py

```diff
@@ -0,0 +1,76 @@
+import json
+import time as _time
+from dataclasses import dataclass
+import binascii
+import collections
+import math
+
+# type: ignore
+from Crypto.PublicKey import RSA  # pylint: disable=import-error
+from Crypto.Random import get_random_bytes  # pylint: disable=import-error
+from Crypto.Signature import pkcs1_15  # pylint: disable=import-error
+from Crypto.Hash import SHA  # pylint: disable=import-error
+from .transaction import Transaction
+
+
+class Node:
+    def __init__(self, value: str, left_child=None, right_child=None):
+        self.value = value
+        self.left_child = left_child
+        self.right_child = right_child
+
+
+class Transactiontree:

```

Comment:
file name change to transaction_tree and change class name change to TransactionTree